src/components/MultiselectTags/MultiselectTags.vue

```diff
@@ -66,7 +66,6 @@ export default {
 <template>
 	<Multiselect :value="inputValue"
 		:options="tags"
-		:options-limit="5"
```

Comment:
I would prefer to keep the ability to limit the tags count and expose it as a prop. If we set the default value of this prop to `undefined` (though I am honestly not sure, this is possible, but let's test), the `slice` method that vue-multiselect uses will just return all tags.

Reply:
Thank's for your quick feedback. To be honest i think that a prop `limit` is not very useful for showing a tag list. The server response would be cut at an arbitrary point, resulting in inconsistent data presented to the user.

If we need a possibility to filter the results received by the server, i'd go with an optional `tagsFilter` prop of type `function` which receives the full tag list and returns a filtered one. A "limt" function could then be easily implementer, too. What do you think?

Reply:
> If we need a possibility to filter the results received by the server, i'd go with an optional `tagsFilter` prop of type `function` which receives the full tag list and returns a filtered one. A "limt" function could then be easily implementer, too. What do you think?

I am not so keen on making this much more complicated than it is now. I just think that the limit was there for a reason and removing it completely might break some uses cases. So keeping the possibility to limit the tag count while allowing to show all seems like a good compromise to me (it will still be a breaking change if we remove the limit by default).

Reply:
I see your point but i really wonder why one should limit the displayed options since there's automatic scrolling happening if there are more options than we can display. 

But ok for compatibility reasons: let's introduce a `limit` prop with default value `5` and let's check if it shows all available options when setting this prop to `null`?

Reply:
> But ok for compatibility reasons: let's introduce a limit prop with default value 5 and let's check if it shows all available options when setting this prop to null?

@raimund-schluessler @skjnldsv would really love to get some feedback on this. I think with that solution we would not break anything and i could realize my usecase for my app so this would be quite important to me. If that's a desirable solution i can change the code accordingly 👍

Reply:
I am not the person in charge of deciding that 😉
But from my side that sounds like a sane approach. I just guess you need to set the prop to `undefined` instead of `null` to get all tags (judging from what I think how `Array.slice` works)